.clang-tidy

```diff
@@ -12,13 +12,21 @@ Checks: |
   -cppcoreguidelines-pro-bounds-pointer-arithmetic
   -cppcoreguidelines-init-variables
   -cppcoreguidelines-pro-type-member-init
+  -cppcoreguidelines-avoid-do-while
+  -cppcoreguidelines-pro-type-const-cast
+  -cppcoreguidelines-pro-bounds-array-to-pointer-decay
+  -cppcoreguidelines-avoid-const-or-ref-data-members
```

Comment:
I don't see these in the output either...

Reply:
also see that locally, e.g.:
```
/home/esseivaj.linux/devel/celeritas/src/celeritas/geo/GeoMaterialParams.cc:142:22: warning: member 'geo_' of type 'const GeoParams &' (aka 'const celeritas::OrangeParams &') is a reference [cppcoreguidelines-avoid-const-or-ref-data-members]
  142 |     GeoParams const& geo_;
```